File: java/test/src/main/java/test/Ice/objects/EI.java
// Copyright (c) ZeroC, Inc.

package test.Ice.objects;

import test.Ice.objects.Test.E;

public final class EI extends E {
    public EI() {
        super(1, "hello");
    }

    public boolean checkValues() {
        return i == 1 && s.equals("hello");
    }
}

File: csharp/test/Ice/facets/EI.cs
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

namespace Ice
{
    namespace facets
    {
        public sealed class E : Test.IE
        {
            public E()
            {
            }

            public string callE(Ice.Current current)
            {
                return "E";
            }
        }
    }
}
